<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictManager</class>
 <widget class="QWidget" name="DictManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>412</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAllButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearDictButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="document-revert">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
